cloudflare/utils.go

```diff
@@ -137,3 +138,17 @@ func stringFromBool(status bool) string {
 	}
 	return "off"
 }
+
+func getAccountID(d *schema.ResourceData, client *cloudflare.API) (string, error) {
+	accountID := d.Get("account_id").(string)
+	if accountID == "" {
+		zoneID := d.Get("zone_id").(string)
+		var err error
```

Comment:
I don't think we need to instantiate `err` here as we're instantiating and assigning in the next line for the `zoneDetails` call.

Reply:
Good catch, this was vestige from before it was pulled into a separate function.